soccer/motion/MotionControl.cpp

```diff
@@ -1,12 +1,12 @@
 #include "MotionControl.hpp"
 
-#include <Context.hpp>
-#include <Geometry2d/Util.hpp>
-#include <Robot.hpp>
-#include <RobotConfig.hpp>
-#include <Utils.hpp>
 #include <optional>
-#include <planning/MotionInstant.hpp>
+
```

Comment:
Trivial changes based on the change in the Trajectory API.

Only exception is that angle targets should be fully resolved by control time, so they are no longer represented by optionals.